src/ansys/additive/core/additive.py

```diff
@@ -213,6 +218,16 @@ def nsims_per_server(self, value: int) -> None:
             raise ValueError("Number of simulations per server must be greater than zero.")
         self._nsims_per_server = value
 
+    @property
+    def enable_beta_features(self) -> bool:
+        """Flag indicating if beta features are enabled."""
+        return self._enable_beta_features
+
+    @enable_beta_features.setter
+    def enable_beta_features(self, value: bool) -> None:
+        """Set the flag indicating if beta features are enabled."""
```

Comment:
You could say: Set the flag to enable beta features.
(But I'm not quite sure what usage you intend here.)

Reply:
This can be "set" to True or False. Setting it to False disables the beta features.

Reply:
ok